tests.py

```diff
@@ -1,15 +1,800 @@
 
 # from CommCareAPIHandler import CommCareAPIHandler, CommCareAPIHandlerPush
 
+import boto3
+import requests
+import json
 import unittest
+import unittest.mock
+from Testing.requests_mock import MockResponse, mock_get, mock_post
+from Testing.boto3_mock import Boto3ClientMock
+from datetime import datetime
+from unittest.mock import MagicMock
+
+def get_boto3_client_mock(Name):
```

Comment:
That makes sense to do!  Do you think it would get confusing though since there is already another file in the repo called util.py that is used by the lambda functions?